doc/source/whatsnew/v0.24.0.txt

```diff
@@ -70,6 +68,7 @@ Current Behavior:
 
 Other Enhancements
 ^^^^^^^^^^^^^^^^^^
+- ``ExcelWriter`` now accepts ``mode`` as a keyword argument, enabling append to existing workbooks when using the ``openpyxl`` engine (:issue:`3441`)
```

Comment:
leave this where it was. this is a significant new feature that's why i put it there.